qutebrowser/commands/runners.py

```diff
@@ -121,10 +121,10 @@ def _get_alias(self, text, default=None):
             otherwise.
         """
         parts = text.strip().split(maxsplit=1)
-        try:
-            alias = config.val.aliases[parts[0]]
```

Comment:
I think that before, this function would throw an error if `text` was empty, and I tried to keep this behavior.